src/config/index.ts

```diff
@@ -12,6 +12,20 @@ export default {
     environment: process.env.NODE_ENV || 'development',
   },
   aws: {
+    sqs: {
+      waitTimeSeconds: 20,
+      batchSize: 10,
+      annotationsDeleteQueue: {
+        url:
+          process.env.SQS_BATCH_DELETE_QUEUE_URL ||
+          'http://localhost:4566/queue/pocket-annotations-delete-queue',
+        visibilityTimeout: 300,
+        maxMessages: 1,
+        waitTimeSeconds: 0,
+        defaultPollIntervalSeconds: 300,
+        afterMessagePollIntervalSeconds: 60,
```

Comment:
I think we want this to be 30 seconds, per the discussion in slack https://pocket.slack.com/archives/C03E28D1GUD/p1657671668758909